lib/frisby.js

```diff
@@ -38,6 +40,7 @@ var globalSetup = function(opts) {
       _frisbyGlobalSetup.request.inspectOnFailure = false;
       _frisbyGlobalSetup.request.json = false;
       _frisbyGlobalSetup.request.baseUri = '';
+      _frisbyGlobalSetup.request.overRideContentType = true;
```

Comment:
Two lines above we're setting json = false... but overRideContentType has the effect of using content-type 'application/json'.  The code isn't telling a good story of what should be expected.